cfme/containers/provider/__init__.py

```diff
@@ -93,6 +93,16 @@ class hawkular(Tab, BeforeFillMixin):  # NOQA
         })
         validate = Button('Validate')
 
+    @View.nested
+    class prometheus(Tab, BeforeFillMixin):  # NOQA
+        TAB_NAME = 'Metrics'
```

Comment:
So there's a bit of confusing interplay between the yaml and these tabs, and it stems from them being called hawkular instead of generically 'metrics'. I think a good update to these modules would be to update the endpoints and these tabs to use metrics, instead of hawkular, so that in the yaml the endpoint can be 'metrics' generically.

If this is removed, the label for the endpoint in the yaml has to be 'hawkular', matching the others.  This would be fixed with updating the nested 'hawkular' view and calling all of the endpoints in the yaml 'metrics'.

Ultimately this new class is being forced in, just to call it 'prometheus', when in reality there is no difference in the CFME UI between the types, only what endpoint is pointed to and how the endpoint is configured/validation.

Unless you would like to take on modifying the other providers, and refactoring this structure a bit to use 'metrics' more generically for the tabs, I think this should be removed. Its not needed to actually support the new endpoint.

Reply:
Well, basicly you right. But I think that fix is out of the scope of this PR.

The main focus of this PR is to allow the FW work with Prometheus metrics as a metrics provider and supply correct configuration before adding an alert endpoint.

I think the solution for the issue you describe here should be in a different PR.